### ISSUE TITLE
RSW: server pod issues warning when adding custom entries to `config.profiles.profiles`

### ISSUE THREAD

**Issue by pat-s** [rstudio/helm#272]
On the server image I see the following warning during startup:

```
[rserver] WARNING Unrecognized option in /mnt/configmap/rstudio/profiles: job-json-overrides = "/spec/template/spec/volumes/-":" │
│ /mnt/job-json-overrides-new/defaultSessionVolume.json","/spec/template/spec/containers/0/volumeMounts/-":"/mnt/job-json-overrides-new/defaultSessionVolumeMount.json │
│ ","/spec/template/spec/volumes/-":"/mnt/job-json-overrides-new/defaultSessionSecretVolume.json","/spec/template/spec/containers/0/volumeMounts/-":"/mnt/job-json-ove │
│ rrides-new/defaultSessionSecretVolumeMount.json"; LOGGED FROM: rstudio::server::user_profiles::UserProfile rstudio::server::user_profiles::{anonymous}::UserProfiles │
│ ::profileFromNode(const ptree&) src/cpp/server/ServerUserProfiles.cpp:438
```

Inspecting `/mnt/configmap/rstudio/profiles` results in 

```
cat /mnt/configmap/rstudio/profiles

[*]
job-json-overrides="/spec/template/spec/volumes/-":"/mnt/job-json-overrides-new/defaultSessionVolume.json","/spec/template/spec/containers/0/volumeMounts/-":"/mnt/job-json-overrides-new/defaultSessionVolumeMount.json","/spec/template/spec/volumes/-":"/mnt/job-json-overrides-new/defaultSessionSecretVolume.json","/spec/template/spec/containers/0/volumeMounts/-":"/mnt/job-json-overrides-new/defaultSessionSecretVolumeMount.json"
session-timeout-kill-hours=24
session-timeout-minutes=60
```

NB: `session-timeout-kill-hours` and `session-timeout-minutes` are set via 

```yml
config:
  profiles:
    profiles:
      "*":
        session-timeout-minutes:
        session-timeout-kill-hours: 	
```

Removing these removes the existence of `/mnt/configmap/rstudio/profiles` entirely, hence not triggering the warning. It looks to me as if the `job-json-overrides` entry there is included by the chart erroneously? Or is my specification of the profiles entries wrong?

**Comment by colearendt** [comment]
Yes this is a bug 😞 #168 

I believe it will also ultimately be resolved by #213 , which should hopefully supersede `job-json-overrides`'s usefulness

**Comment by colearendt** [comment]
Closing in lieu of #168 . Hopefully `job-json-overrides` are less important now, but we can track there until we know for sure
